### PR TITLE
[Owner Review] Added some missing APIs to Digital Pattern

### PR DESCRIPTION
**Pull request by ShantanuShinde**
### What does this Pull Request accomplish?

Added metadata of missing APIs from base metadata and changed `codegen_method` of some non generating APIs from "no" to "public"
115/137 APIs are getting generated currently

### Why should this Pull Request be merged?

The missing APIs generated and built

### What testing has been done?

Build successful and APIs generated

### CODE REVIEW

File: generated/nidigital/nidigital_service.cpp [hunk 1]
```diff
@@ -205,6 +224,8 @@ namespace nidigital_grpc {
       auto vi_grpc_session = request->vi();
       ViSession vi = session_repository_->access_session(vi_grpc_session.id(), vi_grpc_session.name());
       session_repository_->remove_session(vi);
+      auto status = library_->Close(vi);
```
Comment (kruthi-NI) [inline generated/nidigital/nidigital_service.cpp]: @ShantanuShinde this change is not relevant to this PR and can cause confusion for review. So consider undoing this change here. @ni-pkhare  will be sending out a separate PR for this change.

File: source/codegen/metadata/nidigital/functions.py [hunk 2]
```diff
@@ -1427,7 +1427,7 @@
         "returns": "ViStatus"
     },
     "GetChannelName": {
-        "codegen_method": "no",
+        "codegen_method": "public",
```
Comment (kruthi-NI) [inline source/codegen/metadata/nidigital/functions.py:1430]: Document these changes in Changes.md file

File: source/codegen/metadata/nidigital/functions.py [hunk 3]
```diff
@@ -2285,6 +2285,28 @@
...
+            },
+            {
+                "name":"channelList",
+                "direction":"in",
+                "type":"ViConstString"
+            },
+            {
+                "name":"function",
+                "direction":"in",
+                "type":"ViInt32",
```
Comment (kruthi-NI) [inline source/codegen/metadata/nidigital/functions.py:2304]: Mention in PR description that this API which was using ViInt32 was removed earlier in previous PR #64. But now since our feature branch has support for ViInt32, we are adding back this API. Are there any more APIs using ViInt32 which can be added right now?

**Review by kruthi-NI — CHANGES_REQUESTED** [review]
Want to look at Changes.md file once

File: source/codegen/metadata/nidigital/functions.py (hunk 3 above)
Comment (ShantanuShinde) [reply source/codegen/metadata/nidigital/functions.py:2304]: this API got missed when copying the metadata from base to functions.py. so it was added into the PR

File: generated/nidigital/nidigital_service.cpp (hunk 1 above)
Comment (ShantanuShinde) [reply generated/nidigital/nidigital_service.cpp]: removed them

File: source/codegen/metadata/nidigital/functions.py (hunk 2 above)
Comment (ShantanuShinde) [reply source/codegen/metadata/nidigital/functions.py:1430]: I will create separate pr for changes.md and document the changes there

**Review by kruthi-NI — APPROVED** [review]

**Review by jonlugoatx — APPROVED** [review]

**Review by reckenro — APPROVED** [review]
